Makefile

```diff
@@ -15,15 +15,17 @@ MD5CMD = $(shell { command -v md5sum || command -v md5; } 2>/dev/null)
 GUARD = $(1)_GUARD_$(shell echo $($(1)) | $(MD5CMD) | cut -d ' ' -f 1)
 
 FIRSTGOPATH = $(firstword $(subst :, ,$(GOPATH)))
-VERSION_TXT = resources/templates/version.txt
+VERSION_TXT = server/resources/templates/version.txt
```

Comment:
I'm pretty sure I suggested server before, but now I think `signer/` might be more descriptive. Would that change be too annoying?

Reply:
yeah I like that --- especially because in other DLT systems you need a local full/partial "node", which is a near synonym of "server" imho